test-d/types/framework/entity.test-d.ts

```diff
@@ -67,3 +67,24 @@ expectType<Promise<CustomEntity | null>>(CustomEntity.delete(id));
 expectType<Promise<CustomEntity | CustomEntity[] | null>>(CustomEntity.delete([id, id]));
 expectType<Promise<CustomEntity | CustomEntity[] | null>>(CustomEntity.delete([id] as const));
 expectType<Promise<CustomEntity | CustomEntity[] | null>>(CustomEntity.delete([id, id] as const));
+
+type SomeItemData = Item.Data<{}>;
+class SomeItem extends Item<SomeItemData> {}
+type SomeActorData = Actor.Data<
```

Comment:
Maybe you could use a custom class deriving from `Entity` instead of `Actor` (similar to how it's done above). People not knowing what these tests are about might be confused as to why `Actor` is being "tested" here.